README.md

```diff
@@ -176,26 +175,20 @@ newgrp docker
 # Get the project
 git clone https://github.com/pennomi/comics.git
 cd comics
-python3 generate_env.py  # Select "n" because this is a production environment
-docker compose build
 
-# Configure SSL Cert with Let's Encrypt
 # HEADS UP! First remember to set your DNS to the public IP address of the server
-sudo apt install certbot
-sudo certbot certonly --cert-name <project name> --standalone --staple-ocsp -m <your email> --non-interactive --agree-tos -d <example.com> -d <another.example.com>
-
-# HEAD UP! You don't need this yet, but to renew do this.
-certbot renew --post-hook "docker compose restart"
+python3 generate_env.py  # Select "n" because this is a production environment
+docker compose build
 
-# You have the certs, so you can finally start the server
-docker compose up -d
+# Start the server
+docker compose up --build -d
 
 # Create the database and initialize the static files
 docker compose exec django python manage.py migrate
 docker compose exec django python manage.py collectstatic
 
 # EITHER initialize the data
-docker compose exec python manage.py createsuperuser
+docker compose exec django python manage.py createsuperuser

```

Comment:
I feel like corrections like this should be submitted in a separate commit, if not a separate PR. This particular change, for example, is totally unrelated to Caddy, right? It's just a documentation correction?